package.json

```diff
@@ -52,6 +52,10 @@
   "author": "",
   "dependencies": {
     "@hot-loader/react-dom": "^16.8.6",
+    "@types/bn.js": "^4.11.5",
+    "@types/jest": "^24.0.18",
+    "@types/node": "^12.7.4",
```

Comment:
I think there two don't need to be run time dependencies, only test dependencies, no?

Reply:
Right, I don't even remember adding the types of jest or node... it must have been automatic

In this case it can be dev-dependencies cause is not a library, so good call!